Exercism/Gigasecond/Gigasecond.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercism.Gigasecond
+{
+    public static class Gigasecond
+    {
+        public static DateTime Date(DateTime birthDate)
```

Comment:
Also, I think you might consider a different name
```
Gigasecond.Date(date)
```

Doesn't tell me much about what it does. Something like

```
Gigasecond.AddTo(date)
```

or even an extension method


```
date.AddGigasecond()
```

Is much clearer